src/js/popup.js

```diff
@@ -1126,18 +1193,48 @@ Logic.registerPanel(MANAGE_CONTAINERS_PICKER, {
       Logic.showPanel(P_CONTAINER_EDIT, identity);
     };
 
-    document.getElementById("new-container-div").innerHTML = Utils.escaped`
-      <table class="menu">
-        <tr class="menu-item hover-highlight keyboard-nav" id="new-container" tabindex="0">
-          <td>
-            <div class="menu-icon"><img src="/img/new-16.svg" />
-            </div>
-            <span class="menu-text">${ browser.i18n.getMessage("newContainer") }</span>
-          </td>
-        </tr>
-      </table>
-      <hr>
-    `;
+    const divNewContainer = document.getElementById("new-container-div");
+    
+    // Create <table class="menu">
+    const tableMenu = document.createElement("table");
+    tableMenu.classList.add("menu");
+
+    // Create <tr class="menu-item hover-highlight keyboard-nav" id="new-container" tabindex="0">
+    const tableRowMenuItem = document.createElement("tr");
+    tableRowMenuItem.classList.add("menu-item");
+    tableRowMenuItem.classList.add("keyboard-nav");
+    tableRowMenuItem.classList.add("hover-highlight");
+    tableRowMenuItem.id = "new-container";
+    tableRowMenuItem.setAttribute("tabindex", "0");
+
+    // Create <td>
+    const tableData = document.createElement("td");
+    
+    // Create <div class="menu-icon">
+    const divMenuIcon = document.createElement("div");
+    divMenuIcon.classList.add("menu-icon");
+
+    // Create <img src="/img/new-16.svg" />
+    const imgNew = document.createElement("img");
+    imgNew.src = "/img/new-16.svg";
+
+    // Create <span class="menu-text">${ browser.i18n.getMessage("newContainer") }</span>
+    const spanMenuText = document.createElement("span");
+    spanMenuText.classList.add("menu-text");
+    spanMenuText.textContent = browser.i18n.getMessage("newContainer");
+
+    // Create <hr /> 
+    const horizontalRule = document.createElement("hr");
+
+    // Append Children
+    // divNewContainer > ( tableMenu | horizontalRule ) > tableRowMenuItem > tableData > ( divMenuIcon > imgNew ) | spanMenuText
+    divMenuIcon.appendChild(imgNew);
+    tableData.appendChild(divMenuIcon);
+    tableData.appendChild(spanMenuText);
+    tableRowMenuItem.appendChild(tableData);
+    tableMenu.appendChild(tableRowMenuItem);
+    divNewContainer.appendChild(tableMenu);
```

Comment:
At L1236, we need to use `divNewContainer.replaceChildren(tableMenu);` to avoid creating a duplicate of the new container item everytime we enter MANAGE_CONTAINERS_PICKER panel.